frontend/packages/console-shared/src/components/getting-started/__tests__/QuickStartGettingStartedCard.data.tsx

```diff
@@ -0,0 +1,268 @@
+import { QuickStart } from '@console/app/src/components/quick-starts/utils/quick-start-types';
+
+export const loadingQuickStarts: QuickStart[] = [];
+
+export const loadedQuickStarts: QuickStart[] = [
```

Comment:
we already have mock data for quick starts in @console/app

Also, this file name can be `.ts` instead of `.tsx`

Reply:
Changed to `.ts`.

I only see some yaml data, but no mock data which includes the expected featured quick starts. I'm fine with changing this, but I also like to keep this test in depend from others.